crates/ty/tests/cli/rule_selection.rs

```diff
@@ -803,12 +803,12 @@ fn overrides_no_actual_overrides() -> anyhow::Result<()> {
     warning[useless-overrides-section]: Useless `overrides` section
      --> pyproject.toml:5:1
       |
-    3 |   division-by-zero = "error"
+    3 | division-by-zero = "error"
     4 |
-    5 | / [[tool.ty.overrides]]
-    6 | | include = ["*.py"]  # Has patterns but no rule overrides
-      | |__________________^ This overrides section configures no rules
-    7 |   # Missing [tool.ty.overrides.rules] section entirely
+    5 | [[tool.ty.overrides]]
+      | ^^^^^^^^^^^^^^^^^^^^^ This overrides section configures no rules
+    6 | include = ["*.py"]  # Has patterns but no rule overrides
+    7 | # Missing [tool.ty.overrides.rules] section entirely
```

Comment:
TODO: This does not look correct

Reply:
This might be correct, the range has changed from the entire section (including header and content) to only the section header.

Reply:
Yeah, I think it's fine.